crates/router/tests/connectors/stax.rs

```diff
@@ -259,31 +670,99 @@ async fn should_partially_refund_succeeded_payment() {
 // Creates multiple refunds against a payment using the automatic capture flow (Non 3DS).
 #[actix_web::test]
 async fn should_refund_succeeded_payment_multiple_times() {
-    CONNECTOR
-        .make_payment_and_multiple_refund(
+    let customer_response = CONNECTOR
+        .create_connector_customer(customer_details(), get_default_payment_info(None, None))
+        .await
+        .expect("Authorize payment response");
+    let connector_customer_id = match customer_response.response.unwrap() {
+        PaymentsResponseData::ConnectorCustomerResponse {
+            connector_customer_id,
+        } => Some(connector_customer_id),
+        _ => None,
+    };
+
+    let token_response = CONNECTOR
+        .create_connector_pm_token(
+            token_details(),
+            get_default_payment_info(connector_customer_id, None),
+        )
+        .await
+        .expect("Authorize payment response");
+    let payment_method_token = match token_response.response.unwrap() {
+        PaymentsResponseData::TokenizationResponse { token } => Some(token),
+        _ => None,
+    };
+
+    let response = CONNECTOR
+        .make_payment(
             payment_method_details(),
-            Some(types::RefundsData {
-                refund_amount: 50,
-                ..utils::PaymentRefundType::default().0
-            }),
-            get_default_payment_info(),
+            get_default_payment_info(None, payment_method_token.clone()),
         )
-        .await;
+        .await
+        .unwrap();
+
+    //try refund for previous payment
+    let transaction_id = utils::get_connector_transaction_id(response.response).unwrap();
+    for _x in 0..2 {
+        tokio::time::sleep(Duration::from_secs(60)).await; // to avoid 404 error
+        let refund_response = CONNECTOR
+            .refund_payment(
+                transaction_id.clone(),
+                Some(types::RefundsData {
+                    refund_amount: 50,
+                    ..utils::PaymentRefundType::default().0
+                }),
+                get_default_payment_info(None, payment_method_token.clone()),
+            )
+            .await
+            .unwrap();
+        assert_eq!(
+            refund_response.response.unwrap().refund_status,
+            enums::RefundStatus::Success,
+        );
```

Comment:
Why do we need sleep for this unit test alone?

Reply:
We need to wait for a minute to generate a subsequent refund otherwise the connectors throws an error saying:
"The transaction cannot be refunded because there is a similar request currently being processed."